heron_wsgi/templates/kumc_layout.xml

```diff
@@ -3,28 +3,17 @@
 <py:match path="head" once="true">
    ${select('*')}
 
-  <link rel="stylesheet" type="text/css" href=
-  "https://www.kumc.edu/css/template2009/ku-sec.css" />
+  <link rel="stylesheet" type="text/css" href="https://www.kumc.edu/css/template2009/ku-sec.css" />
```

Comment:
i dont see the intention of these particular changes.  was this causing a problem when trying to run the software locally or is this unrelated?

Reply:
Dear @nihr43 this was just me formatting the code to trace a tag that was not properly ended from my code update and also reducing lines of code.